analyses/copy_number_consensus_call/README.md

```diff
@@ -12,8 +12,11 @@ This analysis uses information from the following files generated from the 3 cal
 
 The analysis produces the following output files
 
-* `results/cnv_consensus.tsv`:  A tab separated file out consense copy number variants, including the original calls used for each consensus call
-* `results/pbta-cnv-consensus.seg.gz`: A `.seg` formatted file for downstream processing
+* `results/cnv_consensus.tsv`:  A tab separated file of consensus copy number variants, including the original calls used for each consensus call.
+  *Note: only samples that pass QC are included, see below*
+* `results/uncalled_samples.tsv`: A table of sample-caller pairs excluded from the consensus due to QC failure (usally too many CNV calls, see [Methods: Consensus CNV creation](#consensus-cnv-creation)) If a sample fails QC from two or more callers, it will not appear in downstream files.
```

Comment:
```suggestion
* `results/uncalled_samples.tsv`: A table of sample-caller pairs excluded from the consensus due to QC failure (usually too many CNV calls, see [Methods: Consensus CNV creation](#consensus-cnv-creation)) If a sample fails QC from two or more callers, it will not appear in downstream files.
```